docs/content/jottacloud.md

```diff
@@ -148,9 +148,11 @@ To copy a local directory to an Jottacloud directory called backup
 The official Jottacloud client registers a device for each computer you install it on,
 and then creates a mountpoint for each folder you select for Backup.
 The web interface uses a special device called Jotta for the Archive and Sync mountpoints.
-In most cases you'll want to use the Jotta/Archive device/mountpoint, however if you want to access
-files uploaded by any of the official clients rclone provides the option to select other devices
-and mountpoints during config.
+
+With rclone you'll want to use the Jotta/Archive device/mountpoint in most cases, however if you
+want to access files uploaded by any of the official clients rclone provides the option to select
+other devices and mountpoints during config. Note that uploading files is currently not supported
+to other devices than Jotta.
```

Comment:
What happens if you upload to another device other than Jotta?

Reply:
I was actually a bit surprised that it was blocked in rclone, because when I did other experimenting with Jottacloud API years ago there was no difference. But my guess is that the newer "allocate API" for doing the uploading, introduced here https://github.com/rclone/rclone/pull/2498, did not support it (or at least that is known).

Reply:
You could see if the integration tests pass on a non standard device. If they do they we are good!

Reply:
I can test it out.. Unless @buengese manages to stop me with a good reason not to... :)

Reply:
Found out how we can use the allocate API with other devices than "Jotta", and that seems to work very well: https://github.com/rclone/rclone/pull/5926 (Work in progress)